src/pages/day1.js

```diff
@@ -38,7 +38,6 @@ export const Day1 = () => {
       <Code>cd goker-blog</Code>
```

Comment:
this part and all other code blocks needs to be like this
```
<Code language="bash">{`cd goker-blog`}</Code>
```

remove the <p> tags in Code block and use simply tabs and enters and spaces to align your code samples